shared/shared_defs.h

```diff
@@ -1,6 +1,10 @@
 #ifndef SHARED_DEFS_H
 #define SHARED_DEFS_H
 
+#define SERIAL_BAUD_RATE       BAUD_115200
+
+#define MSG_RECEIVE_TIMEOUT_MS 250
+
 #define ENCODER_COUNTS_PER_REV 500
 #define MOTOR_STEPS_PER_REV    800
```

Comment:
Has the microsteps variable been removed? Is there another place where we may set it.